src/python/pants/engine/internals/mapper.py

```diff
@@ -43,7 +43,7 @@ def parse(
         try:
             target_adaptors = parser.parse(filepath, build_file_content, extra_symbols)
         except Exception as e:
-            raise MappingError(f"Failed to parse {filepath}:\n{e}")
+            raise MappingError(f"Failed to parse ./{filepath}:\n{e}")
```

Comment:
Addresses the confusion described by https://github.com/pantsbuild/pants/issues/12988, so somewhat related to this fix.